adapters/bombcrypto-bnb/index.js

```diff
@@ -0,0 +1,145 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const Binance = require("../../sdk/binance");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+const BigNumber = require("bignumber.js");
+
+class BombCrypto {
+    constructor() {
+        this.name = "bombcrypto-bnb";
+        this.symbol = "BNB";
+        this.token = "bnb";
+        this.protocol = "binance";
+        this.block = 8867059;
+        this.contract = "0xe29f0b490f0d89ca7acac1c7bed2e07ecad65201"; // abi used for 0x5713Ae21F4Bb696A877c90CCcAE310eFF4c4652A(Proxy Contract abi) - https://bscscan.com/address/0xe29F0B490F0d89CA7ACAc1C7BeD2E07eCAD65201#readProxyContract
+        this.events = ["OfferMatched", "ItemListed", "OwnershipTransferred"];
```

Comment:
Are you sure that `ItemListed` and `OwnershipTransferred` events are necessary here? It's pretty self-explanatory, that it's not making any sale here.

Reply:
ItemListed is used to get price in "_getPrice" function
OwnershipTransferred event is used in "getBuyer" function, cause it will be transferred if someone buys the NFT.